apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/RoslynHelpers.cs

```diff
@@ -96,7 +96,7 @@ internal static class RoslynHelpers
         internal static XmlEmptyElementSyntax SeeTag(CrefSyntax cref) =>
             XmlEmptyElement("see").AddAttributes(XmlCrefAttribute(cref));
 
-        internal static SyntaxToken Task() => Identifier("Task");
+        internal static SyntaxToken Task() => Identifier("stt::Task");
```

Comment:
This is no longer just a simple identifier, so this should really use `ParseName(...)` and then the type should be changed to `NameSyntax`, but that causes other problems in some calling code. This should be fine for now, but I'll update in a follow up PR to be correct just in case we reuse these RoslynHelpers in the future in other places.

Reply:
SGTM - I'll add a TODO.